telegram/_forumtopic.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+#
+# A library that provides a Python interface to the Telegram Bot API
+# Copyright (C) 2015-2022
+# Leandro Toledo de Souza <devs@python-telegram-bot.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+"""This module contains objects related to Telegram forum topics."""
+
+from typing import TYPE_CHECKING, Optional
+
+from telegram._telegramobject import TelegramObject
+from telegram._utils.types import JSONDict
+
+if TYPE_CHECKING:
+    from telegram import Bot
+
+
+class ForumTopic(TelegramObject):
+    """
+    This object represents the content of a service message about a new forum topic created in
+    the chat.
+
+    Objects of this class are comparable in terms of equality. Two objects of this class are
+    considered equal, if their :attr:`message_thread_id` and :attr:`name` are equal.
+
+    .. versionadded:: 20.0
+
+    Args:
+        message_thread_id (:obj:`int`): Unique identifier of the forum topic
+        name (:obj:`str`): Name of the topic
+        icon_color (:obj:`int`): Color of the topic icon in RGB format
+        icon_custom_emoji_id (:obj:`str`, optional): Unique identifier of the custom emoji shown
+            as the topic icon.
+
+    Attributes:
+        message_thread_id (:obj:`int`): Unique identifier of the forum topic
+        name (:obj:`str`): Name of the topic
+        icon_color (:obj:`int`): Color of the topic icon in RGB format
+        icon_custom_emoji_id (:obj:`str`, optional): Unique identifier of the custom emoji shown
```

Comment:
Not sure I really should do this. Wherever in Telegram API docs there is a type and "Optional" note, in docstrings in source code I find something like:
```
callback_data (:obj:`str` | :obj:`object`, optional): Data to be sent in a callback query....
```

Reply:
In the Args section we do ``` ... (:obj:`..`, optional) ``` but in Attributes section we do ``` ... (:obj:`...`): Optional. ``` since it renders better imo

Reply:
got it now, thanks. Done.